.travis.yml

```diff
@@ -8,6 +8,7 @@ python:
 before_script:
 - pip install -e .
 - pip install pep8
+- pip install Jinja2
```

Comment:
Since merging develop into this branch, travis-ci failed on "ImportError: No module named jinja2"